app/views/projects.html

```diff
@@ -8,52 +8,201 @@
     <div class="middle-section surface-shaded">
       <div class="middle-container has-scroll">
         <div class="middle-content">
+          <div ng-if="!showGetStarted" class="container-fluid surface-shaded pad-left-none pad-right-none mar-left-none mar-right-none center-block"> <!-- Safari needs surface-shaded at mobile -->
```

Comment:
I think with these little util clases, I'd go the route of making a specific class, then using the utils to modify it in particular instances.  so for example:

```less
// a generic class that works in *most* instances
.object-tile {
  margin: 10px 20px;
  // other rules
}
```
but sometimes needs a tweak:
```html
<!-- in this case, i gotta nix the right margin  -->
<div class="object-tile mar-right-none"></div>
```